sql/logic_test.go

```diff
@@ -207,9 +207,20 @@ func (t *logicTest) setUser(user string) {
 		t.Fatal(err)
 	}
 	certDir := filepath.Join(filepath.Dir(dir), "resource", security.EmbeddedCertsDir)
+
+	certPath := security.ClientCertPath(certDir, user)
+	keyPath := security.ClientKeyPath(certDir, user)
+	// `github.com/lib/pq` requires that private key file permissions are
+	// "u=rw (0600) or less".
+	if err := os.Chmod(keyPath, 0600); err != nil {
```

Comment:
That's annoying. I don't like changing permissions on a checked-in file during a test run, though. Maybe we should copy it to a temp file/directory and set the permissions there.